### PR TITLE
add DocC

### PR DESCRIPTION
**Pull request by oozoofrog**
its progressing.

### CODE REVIEW

**Review by gmlwhdtjd — APPROVED** [review]
It is simply the code to apply the DocC to the package, and I think we need to write more documents.

**Comment by oozoofrog** [comment]
> It is simply the code to apply the DocC to the package, and I think we need to write more documents.

you right, and this is progressing pr. haha.

File: Sources/SwiftLayout/Components/Anchors/Anchors.swift [hunk 1]
```diff
@@ -256,6 +258,128 @@ extension Anchors {
     }
 }
 
+extension Anchors {
+    
+    ///
```
Comment (oozoofrog) [inline Sources/SwiftLayout/Components/Anchors/Anchors.swift:263]: 👍

**Review by oozoofrog — COMMENTED** [review]
hiding sample guide documents until good idea come.

File: Package.swift [hunk 2]
```diff
@@ -1,5 +1,4 @@
-// swift-tools-version:5.4
-// The swift-tools-version declares the minimum version of Swift required to build this package.
+// swift-tools-version:5.5
```
Comment (gmlwhdtjd) [inline Package.swift:1]: Is this the correct version?

**Review by gmlwhdtjd — APPROVED** [review]
How about merging this pr?

File: Package.swift (hunk 2 above)
Comment (oozoofrog) [reply Package.swift:1]: > Is this the correct version?

may be not, it was for DocC, but i think its will be ignored on unsupported version.

Comment (oozoofrog) [inline Package.swift:1]:
```suggestion
// swift-tools-version:5.4
```

**Review by gmlwhdtjd — APPROVED** [review]

**Pull request merged**